server.js

```diff
@@ -0,0 +1,72 @@
+'use strict'
+
+const express = require('express');
+require('dotenv').config();
+
+const app = express();
+app.use(express.static('public'));
+
+const cors = require('cors');
+
+app.use(cors());
+
+const PORT = process.env.PORT || 3000;
+
+
+
+app.listen(PORT, () => console.log(`listening on ${PORT}`));
+
+function Locations (searchQuery, geoDataResults) {
+  this.searchQuery = searchQuery;
+  this.formattedQuery = geoDataResults.results[0].formatted_address;
+  this.latitude = geoDataResults.results[0].geometry.location.lat;
+  this.longitude = geoDataResults.results[0].geometry.location.lng;
+}
+
+
+function Weather (searchQuery, weatherDataResults) {
+  //   this.searchQuery = searchQuery;
+  this.forecast = weatherDataResults.daily.summary;
+  this.time = new Date(weatherDataResults.daily.data[0].time).toDateString();
+  console.log(this);
+}
+
+app.get('/location', (request, response) => {
+  try{
+    let searchQuery = request.query.data;
+    const geoDataResults = require('./data/geo.json');
+    const locations = new Locations(searchQuery, geoDataResults);
+    let correct = false;
+    geoDataResults.results.forEach(city => {
+      if(city.address_components[0].long_name.toLowerCase() === searchQuery.toLowerCase()){
```

Comment:
Geodata results depend on the 'lat' and 'long' you send it, you do not need to validate the response.